persistence/src/main/scala/com/evolutiongaming/akkaeffect/persistence/EventSourcedActorOf.scala

```diff
@@ -1,44 +1,39 @@
 package com.evolutiongaming.akkaeffect.persistence
 
+import java.time.Instant
+
 import akka.actor.Actor
 import akka.persistence.SnapshotSelectionCriteria
-import cats.effect.Async
-import cats.effect.Ref
-import cats.effect.Resource
+import cats.Monad
 import cats.effect.implicits.effectResourceOps
+import cats.effect.{Async, Ref, Resource}
 import cats.syntax.all._
 import com.evolutiongaming.akkaeffect._
-import com.evolutiongaming.catshelper.LogOf
-import com.evolutiongaming.catshelper.ToFuture
-
-import java.time.Instant
-import scala.reflect.ClassTag
+import com.evolutiongaming.akkaeffect.persistence.SeqNr
+import com.evolutiongaming.catshelper.{LogOf, ToFuture}
 
 object EventSourcedActorOf {
 
+  // format: off
   /** Describes lifecycle of entity with regards to event sourcing & PersistentActor Lifecycle phases:
+    * 1. RecoveryStarted: we have id in place and can decide whether we should continue with recovery 
+    * 2. Recovering : reading snapshot and replaying events 
+    * 3. Receiving : receiving commands and potentially storing events & snapshots 
+    * 4. Termination : triggers all release hooks of allocated resources within previous phases
```

Comment:
ScalaDoc will do auto-numeration, if items hand intend (https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html#other-formatting-notes):
```suggestion
    *   1. RecoveryStarted: we have id in place and can decide whether we should continue with recovery 
    *   1. Recovering: reading snapshot and replaying events 
    *   1. Receiving: receiving commands and potentially storing events & snapshots 
    *   1. Termination: triggers all release hooks of allocated resources within previous phases
```

Reply:
This change will be rendered by vscode/intellij incorrectly thus I explicitly set no spaces

Reply:
let's try it over a call, ok?

Reply:
Discussed offline w/ @mr-git : the doc renders correctly in IntelliJ but we still failed to:

- instruct scalafmt not to inline nummered list
- render it properly in VSCode